### ISSUE TITLE
Keyboard Navigation gets stuck in Playback Tool Bar Panel

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#28516]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

using the new speed change feature in the Playback toolbar panel does not seem to be accessible with keyboard navigation.

STEPS:
1.	 Make sure the Playback Toolbar is enabled from the View Menu
2.	From the score press Tab until you get to “Playback Toolbar Panel”.
3.	Press right arrow several times and hear rewind, play, Loop, Metronome, playback settings, blank hours blank, blank minutes blank.
4.	The next right arrow press gets stuck and says blank on all arrow presses in any direction and the only way out seems to be to press Esc.  Closing the speed pop up requires two presses of escape and enter does not select and close as expected.

### Supporting files, videos and screenshots

N/A

### What is the latest version of MuseScore Studio where this issue is present?

4.5.2

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by cbjeukendrup** [comment]
(discussed earlier at https://github.com/musescore/MuseScore/issues/20515, but good to have a separate issue)
